flow360/component/flow360_params/flow360_params.py

```diff
@@ -161,6 +398,14 @@ class Boundary(Flow360BaseModel, metaclass=ABCMeta):
     )
 
 
+class BoundaryWithTurbulenceQuantities(Flow360BaseModel, metaclass=ABCMeta):
+    """Turbulence Quantities on Boundaries"""
+
+    turbulence_quantities: Optional[TurbulenceQuantitiesType] = pd.Field(
+        alias="turbulenceQuantities"
+    )
+
+
```

Comment:
`BoundaryWithTurbulenceQuantities` - better would be to inherit from `Boundary` and in the child classes inherit only from `BoundaryWithTurbulenceQuantities`, not both. When inheriting by 2 we need to remember to not put into conflict these 2 classes. 

Alternatively, if we also want to use `BoundaryWithTurbulenceQuantities` elsewhere (not noly boundaries), the class should be named `TurbulenceQuantities`

Reply:
I thought about it. The reason I did not do that is because then the `BoundaryWithTurbulenceQuantities` would have `type` and `name` property inherited from `Boundary`. 

These new properties will be ignored on the solver side. Just not sure if it will cause confusion to the users.

Anyway I have changed it to inherit from `Boundary`.